common/views.py

```diff
@@ -129,3 +144,117 @@ def post(self, request, *args, **kwargs):
                 messages.error(request, "Invalid token!")
                 return redirect("portal:home")
         return render(request, self.template_name, {"form": form})
+
+
+# @method_decorator(login_required, name="post")
+class EditProfileView(TemplateView):
+    template_name = "user/profile_edit_form.html"
+
+    profile_edit_form_class = ProfileEditForm
+    user_edit_form_class = UserEditForm
+
+    def get_user_profile_info(self, request):
+        # Get the user profile from database
+        try:
+            # Note that not all users have a user profile object
+            # such as the admin superuser
+            user_profile = UserProfile.objects.get(user_id=request.user.id)
+        except ObjectDoesNotExist as e:
+            logger.error(str(e), exc_info=True)
+            user_profile = UserProfile()
+        except Exception as e:
+            logger.error(str(e), exc_info=True)
+            user_profile = UserProfile()
```

Comment:
Btw, just want to tell you that it doesn't solve it for the admin user.
<img width="1256" alt="image" src="https://github.com/user-attachments/assets/b57914a0-cacd-4996-8597-d97b88ce44d6">

Reply:
Added some dummy values for the super user to fix it.